lametro/templates/base.html

```diff
@@ -115,6 +97,8 @@
             {% endblock %}
         </main>
 
+        {% wagtailuserbar 'bottom-left' %}
```

Comment:
Replaces conditional admin link in top nav.

Reply:
Since they'll have to sign in to wagtail to manage the alerts after this, would it be useful to change up the Metro Login link in the footer? Maybe by making it link to the wagtail login instead. That is, as long as logging in through wagtail would still allow them to manage things like manual broadcast links or events that don't appear in legistar, which I imagine it would.

If we think yes, then I can take care of it when I branch off of this!

(Also something to consider in the future is whether wagtail should also somehow manage manual broadcasts.)

Reply:
Oh, good idea. Yes!